test/minikube/minikube_base_database_test.go

```diff
@@ -186,7 +189,7 @@ func restoreDatabase(t *testing.T, namespaceName string) {
 	helm.Install(t, options, testlib.RESTORE_HELM_CHART_PATH, restName)
 	testlib.AddTeardown(testlib.TEARDOWN_RESTORE, func() { helm.Delete(t, options, restName, true) })
 
-	testlib.AwaitPodPhase(t, namespaceName, restName, corev1.PodSucceeded, 120*time.Second)
+	testlib.AwaitPodPhase(t, namespaceName, "restore-demo-", corev1.PodSucceeded, 120*time.Second)
```

Comment:
it took me a while to understand what this name is:
It is `JOBName`-`randomSHA` where `JOBName` is `name: restore-{{ include "restore.target" . }}`.

The SHA is not the same SHA that the Helm chart gets, it is autogenerated by K8s for the particular pod belonging to the JOB.